24). Add topic files 67 to 70 (#17928)
dotnet/docs: docs/standard/linq/write-linq-xml-axis-method.md

---
title: How to write a LINQ to XML axis method - LINQ to XML

> description: You can write your own axis methods to retrieve collections from an XML tree, methods that meet the needs of your applications. Writing extension methods is one of the best ways to do this. The article provides an example for C# and Visual Basic.

Reviewer: ```suggestion
description: Learn how to write your own axis methods to retrieve collections from an XML tree in C# or Visual Basic.
```
